metadata_layer.py

```diff
@@ -352,93 +352,492 @@ def __repr__(self):
         )
 
 class FormatData(object):
-    def __init__(self, content_type, drm_scheme, link=None):
+    def __init__(self, content_type, drm_scheme, link=None, rights_uri=None):
         self.content_type = content_type
         self.drm_scheme = drm_scheme
         if link and not isinstance(link, LinkData):
             raise TypeError(
                 "Expected LinkData object, got %s" % type(link)
             )
         self.link = link
+        self.rights_uri = rights_uri
+
+
+
+class MetaToModelUtility(object):
```

Comment:
I asked Winnie if she thought "DataHorse" would be an apt name, because it's like a workhorse that shuttles data, right?  She didn't think it descriptive :).
I ended up going with "utility" because I'm expecting to put more common code in there in the future.